### PR TITLE
Display names of evolutionary stages in Evolution Tab

### LINKED ISSUE #126: Display names for all evolution stages
**Is your feature request related to a problem? Please describe.**
Evolution tabs only shows sprites.

**Describe the solution you'd like**
Display pokemon names too.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. When mapping over `spriteUrls`, return div with `key` and then add `p` tag under sprite to display Pokemon name.
2. Give `p` tag a class and centered it in `_modal` scss.

## Purpose
Display names in Evolution tab, not just sprites.

## Approach
Create p tag and refactor the `spriteUrls.map()` to return `div` containing `img`(sprite) and `p` tags for each item in array.

## Testing Steps
1. Start app.
2. Open a modal.
3. Go to Evolution tab and check.

Closes #126

### CODE REVIEW

File: src/components/EvolutionTab.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ const EvolutionTab = ({ evolutionChainUrl, pokemonColor }) => {
       }); // end evolutionNameStrings map
 
       Promise.all(urlList).then((data) => {
-        setSpriteUrls(data);
+        setSpriteUrls(data); // data: array of pokemon objects
```
Comment (BlackCubes) [inline src/components/EvolutionTab.js:87]: Nice comment!

**Review by Lap343 — APPROVED** [review]
WOW! so smol I almost didn't see it . Very good Connie

**Review by krferrell — APPROVED** [review]
Looking good. Nice descriptions with the comments

**Review by BlackCubes — APPROVED** [review]
Nice and simple. You're team is going to kill this app!

**Pull request merged**